storm-core/src/clj/backtype/storm/daemon/nimbus.clj

```diff
@@ -15,14 +15,23 @@
 ;; limitations under the License.
 (ns backtype.storm.daemon.nimbus
   (:import [org.apache.thrift.server THsHaServer THsHaServer$Args])
+  (:import [backtype.storm.generated KeyNotFoundException])
+  (:import [backtype.storm.blobstore LocalFsBlobStore])
   (:import [org.apache.thrift.protocol TBinaryProtocol TBinaryProtocol$Factory])
   (:import [org.apache.thrift.exception])
   (:import [org.apache.thrift.transport TNonblockingServerTransport TNonblockingServerSocket])
   (:import [org.apache.commons.io FileUtils])
+  (:import [javax.security.auth Subject])
+  (:import [backtype.storm.security.auth NimbusPrincipal])
   (:import [java.nio ByteBuffer]
            [java.util Collections List HashMap]
            [backtype.storm.generated NimbusSummary])
-  (:import [java.io FileNotFoundException File FileOutputStream])
+  (:import [java.util Iterator])
+  (:import [java.nio ByteBuffer]
+           [java.util Collections List HashMap ArrayList])
```

Comment:
We are importing more than once from `java.util`, `backtype.storm.generated`, `backtype.storm.blobstore`, and possibly some others.  We could consolidate these.  I read that the proper way to do this now is to have a single `:import` per file.  Maybe that would clean things up here.